ext/reflection/tests/bug61388.phpt

```diff
@@ -7,10 +7,6 @@ $x[0] = 'test string 2';
 $x['test'] = 'test string 3';
 $reflObj = new ReflectionObject($x);
 print_r($reflObj->getProperties(ReflectionProperty::IS_PUBLIC));
-
-$x = (object)array("a", "oo" => "b");
-$reflObj = new ReflectionObject($x);
-print_r($reflObj->getProperties(ReflectionProperty::IS_PUBLIC));
```

Comment:
It no longer produces an object with a numeric property, so is not useful to the test. Keeping it around simply demonstrates the normal behaviour of `getProperties`.